### ISSUE TITLE
Method didAddAnnotationViews crashing

### ISSUE THREAD

**Issue by ppamorim** [bigfish24/ABFRealmMapView#14]
Hi.

After some logs of Bugsnag. I noticed that some devices crash on this method. 

    Frameworks/RealmMapView.framework/RealmMapView:0TTSf4g_g_n___TFC12RealmMapView12RealmMapView7mapViewfS0_FTCSo9MKMapView21didAddAnnotationViewsGSaCSo16MKAnnotationView__T_

The full log:

    EXC_BREAKPOINT
    Camerite mh_execute_header
    unknown _mh_execute_header
    Camerite mh_execute_header
    Frameworks/UIKit.framework/UIKit -[UITableView _createPreparedCellForGlobalRow:withIndexPath:willDisplay:]
    Frameworks/UIKit.framework/UIKit -[UITableView _createPreparedCellForGlobalRow:willDisplay:]
    Frameworks/UIKit.framework/UIKit -[UITableView _updateVisibleCellsNow:isRecursive:]
    Frameworks/UIKit.framework/UIKit -[UITableView _performWithCachedTraitCollection:]
    Frameworks/UIKit.framework/UIKit -[UITableView layoutSubviews]
    Frameworks/UIKit.framework/UIKit -[UIView(CALayerDelegate) layoutSublayersOfLayer:]
    Frameworks/QuartzCore.framework/QuartzCore -[CALayer layoutSublayers]
    Frameworks/QuartzCore.framework/QuartzCore _ZN2CA5Layer16layout_if_neededEPNS_11TransactionE
    Frameworks/UIKit.framework/UIKit -[UIView(Hierarchy) layoutBelowIfNeeded]
    Camerite mh_execute_header
    Frameworks/UIKit.framework/UIKit +[UIView(UIViewAnimationWithBlocks) _setupAnimationWithDuration:delay:view:options:factory:animations:start:animationStateGenerator:completion:]
    Frameworks/UIKit.framework/UIKit +[UIView(UIViewAnimationWithBlocks) animateWithDuration:delay:usingSpringWithDamping:initialSpringVelocity:options:animations:completion:]
    Camerite mh_execute_header
    Camerite mh_execute_header
    Camerite mh_execute_header
    Camerite mh_execute_header
    Frameworks/RealmMapView.framework/RealmMapView TTSf4g_g_n___TFC12RealmMapView12RealmMapView7mapViewfS0_FTCSo9MKMapView21didAddAnnotationViewsGSaCSo16MKAnnotationView__T_
    Frameworks/MapKit.framework/MapKit -[MKMapView annotationManager:didSelectAnnotationRepresentation:]
    Frameworks/MapKit.framework/MapKit -[MKAnnotationManager selectAnnotation:animated:avoid:]
    Frameworks/MapKit.framework/MapKit -[MKMapView _handleSelectionAtPoint:]
    Frameworks/MapKit.framework/MapKit -[MKMapView _handleTapToSelect:]
    Frameworks/UIKit.framework/UIKit _UIGestureRecognizerSendTargetActions
    Frameworks/UIKit.framework/UIKit _UIGestureRecognizerSendActions
    Frameworks/UIKit.framework/UIKit -[UIGestureRecognizer _updateGestureWithEvent:buttonEvent:]
    Frameworks/UIKit.framework/UIKit ___UIGestureRecognizerUpdate_block_invoke904
    Frameworks/UIKit.framework/UIKit _UIGestureRecognizerRemoveObjectsFromArrayAndApplyBlocks
    Frameworks/UIKit.framework/UIKit _UIGestureRecognizerUpdate
    Frameworks/CoreFoundation.framework/CoreFoundation __CFRUNLOOP_IS_CALLING_OUT_TO_AN_OBSERVER_CALLBACK_FUNCTION__
    Frameworks/CoreFoundation.framework/CoreFoundation __CFRunLoopDoObservers
    Frameworks/CoreFoundation.framework/CoreFoundation __CFRunLoopRun
    Frameworks/CoreFoundation.framework/CoreFoundation CFRunLoopRunSpecific
    Frameworks/CoreFoundation.framework/CoreFoundation CFRunLoopRunInMode
    PrivateFrameworks/GraphicsServices.framework/GraphicsServices GSEventRunModal
    Frameworks/UIKit.framework/UIKit UIApplicationMain
    Camerite mh_execute_header

The device:

![device_erro](https://cloud.githubusercontent.com/assets/5521682/12262925/5b9d33c6-b912-11e5-9ff8-5586e7544c1a.png)

Thanks!

**Comment by ppamorim** [comment]
I need to upload the dSYM file to make this log more pure.

**Comment by bigfish24** [comment]
Can you provide a repo case, hard to parse these logs?

**Comment by ppamorim** [comment]
@bigfish24 I don't know where's the problem because I cannot upload the dSYM files. I don't have the device that happens the bug too.
My implementation is similar of the sample. I know that is hard to parse the logs like this... :( I have no power here. :(

**Comment by bigfish24** [comment]
What is going on with the table view in the logs? What is going on when the map refreshes the data?

**Comment by lucasromanomr** [comment]
It's happening the same thing in my APP. However I could not play my mistake my device. I noticed that the bug in the iphones: 4s, 5 and 5c to date.
